public/pages/Transforms/utils/metadataHelper.tsx

```diff
@@ -39,7 +41,7 @@ export const renderStatus = (metadata: TransformMetadata | undefined): JSX.Eleme
       break;
     case "init":
       return (
-        <EuiFlexGroup gutterSize="xs">
+        <EuiFlexGroup gutterSize="xs" alignItems={"center"}>
```

Comment:
Nit pick: This may be able to simplify as `alignItems= "center"`.

Reply:
Ah, I'll clean that up in the next revision. I believe the IDE added the curly brackets automatically.